test/modules/aws-cloudwatch-integration.ts

```diff
@@ -34,51 +44,124 @@ describe('AwsCloudwatch Integration Testing', () => {
   `));
 
   it('installs the cloudwatch module', install(modules));
-
-  it('adds a new log group', query(`
+  //
```

Comment:
whats this commented test?

Reply:
Oops. Forgot to uncomment them. It was for me running them in my local computer.